src/dashboard/Data/Browser/ExecuteScriptDialog.react.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016-present, Parse, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the LICENSE file in
+ * the root directory of this source tree.
+ */
+import Modal        from 'components/Modal/Modal.react';
+import React        from 'react';
+import labelStyles  from 'components/Label/Label.scss';
+import styles       from 'dashboard/Data/Browser/ExecuteScriptDialog.scss';
+
+export default class ExecuteScriptDialog extends React.Component {
```

Comment:
Do we need a specific component for this? Is there nothing we can reuse? Or create a generic confirmation dialog component not specific to "script execution"?

Reply:
I have checked the old code. What we have is just a component that is using a generic dialog. We can cut the file out and directly write the content of the file in main file where we are including this file.

But as per code pattern present in the codebase, this is how it is mostly done for similar things.

Reply:
Did what i told, meaning removed it from other file and directly attached it.

Reply:
> But as per code pattern present in the codebase, this is how it is mostly done for similar things.

There may be a good reason for it, not sure. It just doesn't seem efficient to create a new dialog component for every type of dialog. Instead of extending the existing dialog to become more versatile.